content/guides/docker-concepts/running-containers/overriding-container-defaults.md

```diff
@@ -0,0 +1,180 @@
+
+Follow the steps to see how to connect a Redis container to a custom network.
+
+1. Create a network by using the following command:
+
+    ```console
+    $ docker network create mynetwork
+    ```
+
+2. Verify the network by running the following command:
+
+    ```console
+    $ docker network ls
+    ```
+
+3. Connect Redis to the existing network by using the following command:
+
+    ```console
+    $ docker run -d -p 6381:6379 --network mynetwork redis
+    ```
+
+    This will start Redis container in the background, mapped to the host port 6381 and attached to the `mynetwork` network. You passed the `--network` parameter to override the container default by connecting the container to custom Docker network for better isolation and communication with other containers.
+
+    > **Inspecting the container network**
+    >
+    > You can use `docker network inspect` command to see if the container is tied to this new bridge network.
+    { .tip }
+
+### Manage the resources
+
+By default, containers are not limited in their resource usage. However, on shared systems, it's crucial to manage resources effectively. It's important not to let a running container consume too much of the host machine's memory.
+
+This is where the `docker run` command shines again. It offers flags like `--memory` and `--cpus` to restrict how much CPU and memory a container can use.
+
+```console
+$ docker run -d --memory="512m" --cpus=".5" redis
+```
+
+The `--cpus` flag specifies the CPU quota for the container. Here, it's set to half a CPU core (0.5) whereas the `--memory` flag specifies the memory limit for the container. In this case, it's set to 512 MB.
+
+### Override the default `CMD` and `ENTRYPOINT` in Docker Compose
+
+Sometimes, you might need to override the default commands (`CMD`) or entry points (`ENTRYPOINT`) defined in a Docker image, especially when using Docker Compose.
+
+1. Create a `compose.yaml` file with the following content:
+
+    ```yaml
+    services:
+        redis:
+         image: redis
+         entrypoint: ["docker-entrypoint.sh"]
+         command: ["redis-server", "--requirepass", "redispassword"]
+    ```
+
+    The Compose file defines a service named `redis` that uses the official Redis image, sets an entrypoint script, and starts the container with password authentication.
+
+2. Bring up the service by running the following command:
+
+    ```console
+    $ docker compose up -d
+    ```
+
+    This command starts the Redis service defined in the Docker Compose file.
+
+3. Verify the authentication with Docker Dashboard.
+
+    Open the Docker Dashboard, select the **Redis** container and select **Exec** to enter into the container shell.
+
+    ![A screenshot of the Docker Dashboard selecting the Redis container and entering into its shell using EXEC button](images/exec-into-redis-container.webp?border=true) 
+
+## Additional resources
+
+* [Build variables](/build/building/variables/)
```

Comment:
This page doesn't have anything related to build variables so I don't think we need this link. [This page](https://docs.docker.com/engine/reference/run/) has a lot of overlap with this guide so maybe somewhere there is where you want to link to.

Reply:
I have removed the link.